go/common/enclave.go

```diff
@@ -110,6 +110,9 @@ type Enclave interface {
 
 	// DebugTraceTransaction returns the trace of a transaction
 	DebugTraceTransaction(hash gethcommon.Hash, config *tracers.TraceConfig) (json.RawMessage, error)
+
+	// DebugLogVisibility returns the logs of a transaction
+	DebugLogVisibility(hash gethcommon.Hash) (json.RawMessage, error)
```

Comment:
@tudor-malene @BedrockSquirrel Ye ok with this name ? Or do you have sugestions ?

Reply:
Sound 👍